movie-app1.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const selector = document.querySelector('#select-movie')
+    const form = document.querySelector('#form')
+
+    selector.addEventListener('change', () => {
+       if (mainDiv === '') {
+           displayMovieContent(selector.selectedIndex - 1)
+       } else {
+           displayMovieContent(selector.selectedIndex - 1)
+       }
+    }) 
+    form.addEventListener('submit', postReview)
+})
+
+const mainDiv = document.querySelector('#maincontent')
+const optionVal = document.querySelector('option')
+const mainURL = "https://ghibliapi.herokuapp.com/films"
+
+const grabMovies = async () => {
+    const movieRequest = await axios.get(mainURL)
+    const movieInfo = movieRequest.data
+    console.log(movieInfo)
+    createMovieOptions(movieInfo)
+}
+grabMovies()
+
+const createMovieOptions = (movies) => {
+    const selector = document.querySelector('#select-movie')
+    let index = 1
+    for (let i = 0; i < movies.length; i++){
+        const option = document.createElement('option')
+        option.text = movies[i].title
+        option.id = index
+        selector.add(option)
+        index++
+    }
+}
+
+const displayMovieContent = async (optionId) => {
```

Comment:
This function works as intended but you might want to add removeChild method somewhere since it is adding movie info without removing info for previous movie.